database.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
It would be more secure to use environment variables for the database credentials instead of hardcoding them.